README.md

```diff
@@ -1,38 +1,97 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+<div align="center">
 
-## Getting Started
+<svg width="150" height="150" viewBox="0 0 42 43" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Maybe is more interesting to create a folder for readme assets(.github/assets maybe?) to save this SVG. This approach improves readability.